units/host_status.go

```diff
@@ -176,6 +164,13 @@ func (j *cloudHostReadyJob) terminateUnknownHosts(ctx context.Context, awsErr st
 		if h == nil {
 			continue
 		}
+		// Decommission the host to prevent this job from checking it again.
+		grip.Error(message.WrapError(h.SetDecommissioned(evergreen.User, false, "cloud host has no status"), message.Fields{
```

Comment:
why do we want to log this here instead of adding to the catcher?

Reply:
I guess it doesn't really matter, since both end up just being logged as errors anyway. I changed it to catch the error.